pkg/kubelet/cri/remote/remote_runtime.go

```diff
@@ -67,7 +71,7 @@ const (
 )
 
 // NewRemoteRuntimeService creates a new internalapi.RuntimeService.
-func NewRemoteRuntimeService(endpoint string, connectionTimeout time.Duration) (internalapi.RuntimeService, error) {
+func NewRemoteRuntimeService(endpoint string, tp *trace.TracerProvider, connectionTimeout time.Duration) (internalapi.RuntimeService, error) {
```

Comment:
It is a little weird to be inserting tp as the middle argument, can you move that to the end?

Reply:
updated, moved to end